<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simUI</class>
 <widget class="QMainWindow" name="simUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="consoleScreen">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>simUI Console</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Simulation_Source"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_simUI"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCPU_Specific_Debugging_Information"/>
    <addaction name="actionRegister_Inspector"/>
    <addaction name="actionMemory_Inspector"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <widget class="QMenu" name="menuBreakpoints">
     <property name="title">
      <string>Breakpoints</string>
     </property>
     <addaction name="actionAdd_Breakpoint"/>
     <addaction name="actionList_Current_Breakpoints"/>
    </widget>
    <addaction name="actionStart_Simulation"/>
    <addaction name="actionStop_Simulation"/>
    <addaction name="separator"/>
    <addaction name="menuBreakpoints"/>
    <addaction name="actionBreak_Execution"/>
    <addaction name="actionRuntime_Directive"/>
    <addaction name="separator"/>
    <addaction name="actionCPU_Options"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionClear_Console_Window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Simulation Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart_Simulation"/>
   <addaction name="actionStop_Simulation"/>
   <addaction name="actionBreak_Execution"/>
   <addaction name="actionRuntime_Directive"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Breakpoint"/>
   <addaction name="actionList_Current_Breakpoints"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionAbout_simUI">
   <property name="text">
    <string>About simUI...</string>
   </property>
  </action>
  <action name="actionRegister_Inspector">
   <property name="text">
    <string>Register Inspector</string>
   </property>
  </action>
  <action name="actionMemory_Inspector">
   <property name="text">
    <string>Memory Inspector</string>
   </property>
  </action>
  <action name="actionCPU_Specific_Debugging_Information">
   <property name="text">
    <string>CPU Specific Debugging Information</string>
   </property>
  </action>
  <action name="actionStart_Simulation">
   <property name="icon">
    <iconset resource="simUIart.qrc">
     <normaloff>:/icons/art/media-playback-start.png</normaloff>:/icons/art/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </action>
  <action name="actionStop_Simulation">
   <property name="icon">
    <iconset resource="simUIart.qrc">
     <normaloff>:/icons/art/media-playback-stop.png</normaloff>:/icons/art/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Simulation</string>
   </property>
  </action>
  <action name="actionBreak_Execution">
   <property name="icon">
    <iconset resource="simUIart.qrc">
     <normaloff>:/icons/art/media-playback-pause.png</normaloff>:/icons/art/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Break Execution</string>
   </property>
  </action>
  <action name="actionRuntime_Directive">
   <property name="icon">
    <iconset resource="simUIart.qrc">
     <normaloff>:/icons/art/debug-run.png</normaloff>:/icons/art/debug-run.png</iconset>
   </property>
   <property name="text">
    <string>Runtime Directive</string>
   </property>
  </action>
  <action name="actionSet_Simulation_Source">
   <property name="text">
    <string>Set Simulation Source</string>
   </property>
  </action>
  <action name="actionAdd_Breakpoint">
   <property name="text">
    <string>Add Breakpoint</string>
   </property>
  </action>
  <action name="actionList_Current_Breakpoints">
   <property name="icon">
    <iconset resource="simUIart.qrc">
     <normaloff>:/icons/art/flag-red.png</normaloff>:/icons/art/flag-red.png</iconset>
   </property>
   <property name="text">
    <string>List Current Breakpoints</string>
   </property>
  </action>
  <action name="actionCPU_Type_Selection">
   <property name="text">
    <string>CPU Type Selection</string>
   </property>
  </action>
  <action name="actionMemory_Size_Selection">
   <property name="text">
    <string>Memory Size Selection</string>
   </property>
  </action>
  <action name="actionCPU_Options">
   <property name="text">
    <string>Simulation Options</string>
   </property>
  </action>
  <action name="actionClear_Console_Window">
   <property name="text">
    <string>Clear Console Window</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simUIart.qrc"/>
 </resources>
 <connections/>
</ui>
